pkg/auth/webapp/csrf.go

```diff
@@ -21,3 +27,44 @@ func NewCSRFCookieDef(cfg *config.HTTPConfig) CSRFCookieDef {
 
 	return def
 }
+
+type CSRFDebugMiddleware struct {
+	Cookies CookieManager
+}
+
+var CSRFDebugCookieMaxAge = int(duration.Short.Seconds())
```

Comment:
Use duration.UserInteraction, same as the actual CSRF cookie.

Reply:
ok